### PR TITLE
add advanced lab 2

### PR DESCRIPTION
**Pull request by glee-**

### CODE REVIEW

**Comment by glee-** [comment]
pr split into two distinct commits

**Comment by glee-** [comment]
to be merged when the vms are provisioned

File: labs/a2.md [hunk 1]
```diff
@@ -0,0 +1,186 @@
...
+
+### Exercise 2: Troubleshooting
+Now we are going to try and troubleshoot a package. Move to the other folder, `ex2`.
+
+Try installing the `ocfspy` package using `dpkg`. It should error. Take note what it is erroring on! Now try and fix it.
+
+Hint: Inspect the package for more details. The file to create that application is in the folder. Try compiling and packaging it. Exercise 1 may be a useful reference if you are stuck.
+
+### Conclusion
+After you’re done, complete [this](https://goo.gl/forms/WMhdZXAcTDPG7gOW2) submission form.
```
Comment (dkess) [inline labs/a2.md]: Please add the feedback questions at the end of the lab.

Comment (glee-) [reply labs/a2.md]: Feedback questions are on page 2 of the submission form

**Pull request merged**